ocfweb/docs/docs/staff/staffvm/ocfweb.md

```diff
@@ -0,0 +1,26 @@
+[[!meta title="Running ocfweb on your Staff VM"]]
+
+If you want to work on [`ocfweb`][0] on your Staff VM, it isn't as
+simple as cloning the repo and running `make dev`, unfortunately.
+In order to do so, you will need to add the `ocfweb` dev config to
+your Staff VM's Puppet configuration, so Puppet can install
+`ocfweb`'s dependencies. To do this, clone the [puppet repo][1] or
```

Comment:
It's not really about dependencies, actually those still need to be installed manually (PRs on this are welcome), but instead it adds config files with secrets like the password to connect to redis on supernova, and the password to connect to the `ocfmail` database. [The manifest](https://github.com/ocf/puppet/blob/master/modules/ocf_ocfweb/manifests/dev_config.pp) really doesn't do much besides provide a few files.